### DISCUSSION TITLE
Add option to play beyond In and Out

### DISCUSSION THREAD

**Discussion by robodean** [videovillage/community#105] [Ideas]
Currently, when in and out markers are set, playback loops within them. 
I would humbly offer that this should not be the default, but rather an optional mode. 
Default should be normal playback, from whatever position in the timeline you happen to be in.
In this case, SHIFT+PLAY would loop between the markers. 

cheers

**Comment by gkertai** [comment]
+1 on this.

I've come across this limitation many times. The current handling of in/out for playback makes it difficult to check whether I set my in/out points correctly without removing them. Eg. I would need to see what the next few seconds are after my out point to make sure those need to be excluded. This is currently quite cumbersome to do, the only way is to mouse-scrub, and the longer the video, the harder it gets.
